DataAccessLayer/Crud/Implementation/LanguageCrud.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DataAccessLayer.DataBaseModels;
+using DataAccessLayer.Crud.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccessLayer.Crud.Implementation
+{
+    public class LanguageCrud : ICrudDictionary<Language>
+    {
+        public List<Language> GetAll()
+        {
+            try
+            {
+                using (LanguageSkillsDBContext db = new LanguageSkillsDBContext())
+                {
+                    return db.Languages.ToList();
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return new List<Language>();
+            }
+        }
+
+        public Language Get(int id)
+        {
+            try
+            {
+                using (LanguageSkillsDBContext db = new LanguageSkillsDBContext())
+                {
+                    return db.Languages.Find(id);
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                return new Language();
+            }
+        }
+
+        public void CreateRange(List<Language> languages)
+        {
+            try
+            {
+                using (LanguageSkillsDBContext db = new LanguageSkillsDBContext())
+                {
+                    db.Languages.AddRange(languages);
+                    db.SaveChanges();
+                }
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+            }
+        }
+
+        public void Update(Language language)
+        {
+            try
+            {
+                using (LanguageSkillsDBContext db = new LanguageSkillsDBContext())
+                {
+                    db.Entry(language).State = EntityState.Modified;
```

Comment:
Did you check that this approach works OK and all changes are saved?
Please check and fix if required.

Also relevant to all other CRUD classes